runtime/ftplugin/man.vim

```diff
@@ -4,8 +4,10 @@ endif
 let b:did_ftplugin = 1
 
 if expand('%') !~# '^man:\/\/'
-  call man#normalize_page()
-  silent execute 'file '.'man://'.tolower(matchstr(getline(1), '^\S\+'))
+  " remove all those backspaces
+  silent execute 'keepjumps %substitute,.\b,,ge'
```

Comment:
Will fix, not quite sure why execute is there actually. I think its an artifact from something funky I was doing when experimenting with the best way to handle the backspaced text.